src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -1007,4 +1007,178 @@ object SampleWdl {
 
     override val rawInputs =  Map.empty[String, String]
   }
+
+  class ScatterWdl extends SampleWdl {
+    val tasks = """task A {
+      |  command {
+      |    echo -n -e "jeff\nchris\nmiguel\nthibault\nkhalid\nscott"
+      |  }
+      |  output {
+      |    Array[String] A_out = read_lines(stdout())
+      |  }
+      |}
+      |
+      |task B {
+      |  String B_in
+      |  command {
+      |    python -c "print(len('${B_in}'))"
+      |  }
+      |  output {
+      |    Int B_out = read_int(stdout())
+      |  }
+      |}
+      |
+      |task C {
+      |  Int C_in
+      |  command {
+      |    python -c "print(${C_in}*100)"
+      |  }
+      |  output {
+      |    Int C_out = read_int(stdout())
+      |  }
+      |}
+      |
+      |task D {
+      |  Array[Int] D_in
+      |  command {
+      |    python -c "print(${sep='+' D_in})"
+      |  }
+      |  output {
+      |    Int D_out = read_int(stdout())
+      |  }
+      |}
+      |
+      |task E {
+      |  command {
+      |    python -c "print(9)"
+      |  }
+      |  output {
+      |    Int E_out = read_int(stdout())
+      |  }
+      |}
+    """.stripMargin
+
+    override def wdlSource(runtime: String = "") =
+      s"""$tasks
+        |
+        |workflow w {
+        |  call A
+        |  scatter (item in A.A_out) {
+        |    call B {input: B_in=item}
+        |    call C {input: C_in=B.B_out}
+        |    call E
+        |  }
+        |  call D {input: D_in=B.B_out}
+        |}
+      """.stripMargin
+
+    override lazy val rawInputs = Map.empty[String, String]
+  }
+
+  object SiblingsScatterWdl extends ScatterWdl {
+    override def wdlSource(runtime: String = "") =
+      s"""$tasks
+        |
+        |workflow w {
+        |  call A
+        |  scatter (item in A.A_out) {
+        |    call B {input: B_in=item}
+        |    call C {input: C_in=B.B_out}
+        |    call E
+        |  }
+        |  scatter (item in A.A_out) {
+        |    call B as F {input: B_in=item}
+        |  }
+        |  call D {input: D_in=B.B_out}
+        |}
+      """.stripMargin
+
+    override lazy val rawInputs = Map.empty[String, String]
+  }
+
+  object SimpleScatterWdl extends SampleWdl {
+    override def wdlSource(runtime: String = "") =
+      """task echo_int {
+        |  Int int
+        |  command {echo ${int}}
+        |  output {Int out = read_int(stdout())}
+        |}
+        |
+        |workflow scatter0 {
+        |  Array[Int] ints = [1,2,3,4,5]
+        |  call echo_int as outside_scatter {input: int=8000}
+        |  scatter(i in ints) {
+        |    call echo_int as inside_scatter {
+        |      input: int=i
+        |    }
+        |  }
+        |}
+      """.stripMargin
+
+    override lazy val rawInputs = Map.empty[String, String]
+  }
+
+  object DockerScatterWdl extends SampleWdl {
```

Comment:
Bleh, I see it's all in the docs and everything. I'd prefer that we not encourage the "prepare scatter gather" terminology, at least at an official level. 

I suppose it's fine as long as it's clear that they're trite examples and not patterns that people should feel any desire to follow.